DiscordManimator.py

```diff
@@ -104,29 +104,52 @@ async def manimate(ctx, *, arg):
                 container_stderr = dockerclient.containers.run(
                     image="manimcommunity/manim:stable",
                     volumes={tmpdirname: {'bind': '/manim/', 'mode': 'rw'}},
-                    command=f"timeout 120 manim /manim/script.py -qm -o scriptoutput {cli_flags}",
+                    command=f"timeout 120 manim /manim/script.py -qm --disable_caching -o scriptoutput {cli_flags}",
                     user=os.getuid(),
                     stderr=True,
                     stdout=False,
                     remove=True
                 )
                 if container_stderr:
-                    await ctx.reply("Something went wrong, here is "
-                                    "what Manim reports:\n"
-                                    f"```\n{container_stderr.decode('utf-8')}\n```")
+                    if len(container_stderr.decode('utf-8')) <= 1200:
+                        await ctx.reply("Something went wrong, here is "
+                                        "what Manim reports:\n"
+                                        f"```\n{container_stderr.decode('utf-8')}\n```")
+                    else:
+                        await ctx.reply("Something went wrong, here is "
+                                        "what Manim reports:\n",
+                                        file = discord.File(
+                                            fp=BytesIO(container_stderr),
+                                            filename="Error.log"
+                                            )
+                                        )
                     return
 
             except Exception as e:
                 await ctx.reply(f"Something went wrong: ```{e}```")
                 raise e
 
             try:
+                reply = None
                 [outfilepath] = Path(tmpdirname).rglob('scriptoutput.*')
-                await ctx.reply("Here you go:", file=discord.File(outfilepath))
+                reply = await ctx.reply("Here you go:", file=discord.File(outfilepath))
+
             except Exception as e:
                 await ctx.reply("Something went wrong: no (unique) output file was produced. :cry:")
+                raise e
```

Comment:
I would prefer for the try/catch block to be focused more on the `rglob` call, as I *assume* that's the thing that raises the error that makes `reply` not be set, so something like
```py
reply = None

try:
    [outfilepath] = Path(tmpdirname).rglob("scriptoutput.*")
except Exception as e:
    ...

reply = ...
```

Though now looking at it closer, it seems to me that `if reply:` wil only be checked if an error doesn't occur, which makes the `reply = None` and checking that redundant because the exception is re-raised when it's caught

Reply:
We might need to "modularise" the `reply` and `if reply` anyway so we can let other messages, not just successful renders be deleted. I think we can do that after this PR.

Reply:
Well, I'd rather not have weird/hard to read code in the main branch at any point, and one of the problems with try/except blocks is that it gets really hard to track what could interrupt the code flow in the try block, so generally I like to keep them *really* short and sweet

Reply:
Done in latest commit.